code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -591,6 +570,24 @@
 		return
 
 	diseasescan(user, patient, src)
+//Checks the individual for any diseases that are visible to the scanner, and displays the diseases in the attacked to the attacker.
+/proc/diseasescan(mob/user, mob/living/carbon/patient, obj/item/healthanalyzer/wound/scanner)
+	if(!istype(patient) || user.incapacitated())
+		return
+
+	var/list/render
```

Comment:
```suggestion
	var/list/render = list()
```
lists need a like, list to point to
otherwise they just default to null (types are just compile hints and initial memes)

this code only works because null += "text" -> "text"